src/utils/util/requirements.txt

```diff
@@ -1,2 +1,3 @@
 jsonschema
 xmlschema
+pyyaml
```

Comment:
tl;dr: can we stick with `ruamel.yaml` or will that be a chore? 

I am the original author of usnistgov/OSCAL#1070. On a similar note I [chose the `ruamel` YAML runtime](https://github.com/usnistgov/OSCAL/pull/1070/files#diff-ab47424d799644e759937769e248b6340fb5412618159e4e4d092530299d0260R10) for reasons around [extensibility and improvements around YAML handling in contrast to `pyyaml`](https://yaml.readthedocs.io/en/latest/pyyaml.html#api). However I think `ruamel.yaml` does not inherently support the `validate` function, it would be great if you or I (once I have time) check if we can align on the one library. I had a lot of problems getting `pyyaml` to work for complex stuff and requires [binding with native C `libyaml` project where is with `ruamel.yaml` it's optional](https://yaml.readthedocs.io/en/latest/install.html#optional-requirements). Makes it behave better in different environments (especially for CI/CD).

Reply:
@aj-stein-nist I have no problem using `ruamel.yaml` instead of `pyyaml`. The reason I chose `pyyaml` is due to the fact that it's the most popular YAML Python library, but I have no strong opinion either way.

Regarding support for `validate` function, since YAML schemas for OSCAL don't yet exist (at least not that I could see) I'm using jsonschema validation on the loaded YAML object to validate the YAML files.

Reply:
I'm sorry, I misread the function signature completely. I will complete review of this PR shortly for final feedback and hopefully a merge soon! :-)